server/poetry.lock

```diff
@@ -799,7 +799,7 @@ python-versions = ">=2.6, !=3.0.*, !=3.1.*, !=3.2.*"
 version = "2.4.5"
 
 [[package]]
-category = "dev"
+category = "main"
```

Comment:
This was done to build only one image in CI instead of two, one with pytest and one without it, as this is quiet time consuming.